README.md

```diff
@@ -71,471 +42,22 @@ for an introduction to our philosophy of pipeline data quality testing.
-					Prefect
-				</a>
-			</td>
-			<td>
-				Orchestrator
-			</td>
-		</tr>
-		<tr>
-			<td style="text-align: center; height: 40px; background-color: #fff;">
-				<img height="40" src="./docs/readme_assets/zenml.jpg" />
-			</td>
-			<td style="width: 200px;">
-				<a href="https://docs.greatexpectations.io/docs/oss/integrations/integration_zenml/">
-					ZenML
-				</a>
-			</td>
-			<td>
-				Orchestrator
-			</td>
-		</tr>
-		<tr>
-			<td style="text-align: center; height: 40px; background-color: #fff;">
-				<img height="40" src="./docs/readme_assets/slack.jpg" />
-			</td>
-			<td style="width: 200px;">
-				<a href="https://docs.greatexpectations.io/docs/oss/guides/validation/validation_actions/how_to_trigger_slack_notifications_as_a_validation_action/">
-					Slack
-				</a>
-			</td>
-			<td>
-				Plugin
-			</td>
-		</tr>
-	</tbody>
-</table>
-
-
-What is GX _not_?
--------------------------------------------------------------
-
-Great Expectations is _not_ a pipeline execution framework. Instead, it integrates seamlessly with DAG execution tools
-like [Spark]( https://spark.apache.org/), [Airflow](https://airflow.apache.org/), [dbt]( https://www.getdbt.com/)
-, [prefect](https://www.prefect.io/), [dagster]( https://github.com/dagster-io/dagster)
-, [Kedro](https://github.com/quantumblacklabs/kedro), [Flyte](https://flyte.org/), etc. GX carries out your data quality
-pipeline testing while these tools execute the pipelines.
-
-Great Expectations is _not_ a database or storage software. It processes your data in place, on your existing systems.
-Expectations and Validation Results that GX produces are metadata about your data.
-
-Great Expectations is _not_ a data versioning tool. If you want to bring your data itself under version control, check
-out tools like [DVC](https://dvc.org/), [Quilt](https://github.com/quiltdata/quilt),
-and [lakeFS](https://github.com/treeverse/lakeFS/).
-
-Great Expectations is _not_ a language-agnostic platform. Instead, it follows the philosophy of “take the compute to the
-data” by using the popular Python language to support native execution of Expectations in pandas, SQL (via SQLAlchemy),
-and Spark environments.
-
-Great Expectations is _not_ exclusive to Python programming environments. It can be invoked from the command line
-without a Python environment. However, if you’re working into another ecosystem, you may want to explore
-ecosystem-specific alternatives such as [assertR](https://github.com/ropensci/assertr) (for R environments)
-or [TFDV](https://www.tensorflow.org/tfx/guide/tfdv) (for Tensorflow environments).
-
+## Get started
 
-Who maintains Great Expectations?
--------------------------------------------------------------
+Visit [Get started with Great Expectations](https://docs.greatexpectations.io/docs/oss/guides/setup/get_started_lp).
 
-Great Expectations OSS is under active development by GX Labs and the Great Expectations community.
+## Get support
```

Comment:
@joshzzheng we can certainly provide the installation instructions (`pip install great_expectations`), but that does nothing for users. They need to create a Data Context and then connect to a Data Source. This is a complex process that is better described in our documentation. I don't think we want to duplicate our so-called quickstart here as it will add a maintenance burden. Linking to documentation and allowing users to choose their own path is the better option and also promotes the documentation. If you want some of the content from our quickstart added here, let me know. However, be aware that it won't help users get started.